lib/dbservice_web/controllers/student_controller.ex

```diff
@@ -76,7 +76,7 @@ defmodule DbserviceWeb.StudentController do
   def create(conn, params) do
     case Users.get_student_by_student_id(params["student_id"]) do
       nil ->
-        create_new_student(conn, params)
+        create_student_with_user(conn, params)
 
       existing_student ->
         update_existing_student(conn, existing_student, params)
```

Comment:
This should be `update_existing_student_with_user`

Reply:
It's only updating student. This is still the same as before.

Reply:
That's what I'm saying. It shouldn't only update student. Rather it should now update student along with its user!

Reply:
Okay, read the next reviews and got it.

Reply:
So, let's make the changes then?